playwright-e2e/utils/faker-utils.ts

```diff
@@ -17,3 +19,13 @@ export const generateEmailAlias = (email: string | undefined): string => {
   const domain = splintedEmail[1];
   return `${name}+${Math.floor(Math.random() * 1000000000)}@${domain}`;
 };
+
+export const setPatientParticipantGuid = async (page: Page) => {
+  const [response] = await Promise.all([
+    page.waitForResponse((resp) => resp.url().includes('/participants') && resp.status() === 200)
+  ]);
+
+  const participantResponse = response.url();
+  const urlArray = participantResponse.split('/');
+  user.patient.participantGuid = urlArray[6];
+};
```

Comment:
Waiting and parsing a request in a separate function is going to make tests flaky because request could have been finished before this function starts to run. Best way to do this is, perform action and wait for response together a in Promise.all.

```
await Promise.all([
  some_action(),
  waitForResponse()
]);
```

Reply:
will do - thinking a bit more about this one - in terms of which action to use - will make the changes